### PR TITLE
Add Compose File and Shell Script for Simplified ODE Setup

### LINKED ISSUE #1: Automated Setup for Local ODE with Docker
The minimally required components of the Operational Data Environment should be able to run with little to no user configuration. A script can be provided to simplify the use of Docker for this local deployment.

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
Users need access to an ODE instance for generating messages during Inter-Op testing, but setting up and configuring the ODE can be complex.

## Solution
A compose file and a shell script have been provided to simplify the process of starting the ODE. Instructions for using these resources have been added to the README.

## Relevant Issue
Closes #1

### CODE REVIEW

File: start_ode.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          export DOCKER_HOST_IP=$(ifconfig | grep -A 1 'inet ' | grep -v 'inet6\|127.0.0.1' | awk '{print $2}' | grep -E '^172\.1[6-9]\.|^172\.2[0-9]\.|^172\.3[0-1]\.|^192\.168\.' | head -n 1)
+      fi
+      if [ -z $DOCKER_HOST_IP ]
+      then
+          echo "DOCKER_HOST_IP is not set and could not be determined. Exiting."
+          exit 1
+      fi
+
+      # write DOCKER_HOST_IP to .env
+      echo "DOCKER_HOST_IP=$DOCKER_HOST_IP" > .env
```
Comment (payneBrandon) [inline start_ode.sh:33]: Do we need to create the .env file first?

Comment (dmccoystephenson) [reply start_ode.sh:33]: We do not. Writing output to a file like this creates the file if it doesn't already exist and overwrites it if it does.

Comment (payneBrandon) [reply start_ode.sh:33]: ope, looks like writing to it also creates it so we should be good to go

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**